src/pages/token/rewards/UnbondingPanels.tsx

```diff
@@ -0,0 +1,119 @@
+import { shallowEqual } from 'react-redux';
+import styled from 'styled-components';
+import { formatUnits } from 'viem';
+
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStakingValidator } from '@/hooks/useStakingValidator';
+import { useStringGetter } from '@/hooks/useStringGetter';
+import { useTokenConfigs } from '@/hooks/useTokenConfigs';
+
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { AssetIcon } from '@/components/AssetIcon';
+import { Output, OutputType } from '@/components/Output';
+import { Panel } from '@/components/Panel';
+import { ValidatorFaviconIcon } from '@/components/ValidatorName';
+
+import { getUnbondingDelegations } from '@/state/accountSelectors';
+import { useAppSelector } from '@/state/appTypes';
+
+export const UnbondingPanels = () => {
+  const stringGetter = useStringGetter();
+  const unbondingDelegations = useAppSelector(getUnbondingDelegations, shallowEqual);
+  const { unbondingValidators } = useStakingValidator() ?? {};
+  const { chainTokenLabel, chainTokenDecimals } = useTokenConfigs();
+
+  if (!unbondingDelegations?.length) {
+    return null;
+  }
+
+  return (
+    <$Container>
+      {unbondingDelegations.map((delegation) => {
+        const completionDate = new Date(delegation.completionTime).getTime();
+        const currentDate = new Date().getTime();
+        const timeDifference = completionDate - currentDate;
+
+        const dayDifference = Math.floor(timeDifference / (1000 * 60 * 60 * 24));
```

Comment:
[q] do we have a const for this?

Reply:
does this return `new Date() - new Date(delegation.completionTime)` return a `Duration` by any chance? if so we should just be able to use `result.days`

Reply:
I think it's just a number (milliseconds)

Reply:
good call, we actually do have a const for this